src/sqs/queues.ts

```diff
@@ -22,7 +22,7 @@ const sqsQueues = {
 		queueRegion: envVars.SQS_PUSH_QUEUE_REGION,
 		longPollingIntervalSec: LONG_POLLING_INTERVAL_SEC,
 		timeoutSec: 60,
-		maxAttempts: 5}, pushQueueMessageHandler,	defaultErrorHandler),
+		maxAttempts: 5}, pushQueueMessageHandler,	pushQueueErrorHandler),
```

Comment:
We don't need a feature flag for this change because if something goes wrong we can always use existing feature flag to fallback to the Redis queue.